steps/autify-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Autify Upload
+summary: |
+  Upload a build file to Autify for Mobile.
+description: |
+  [Autify for Mobile](https://autify.com/mobile) is a test automation platform for your native app.
+  This step allows you to upload your app built in the previous steps to Autify.
+
+  You need to build your app first by `Android Build`, `Xcode build for simulator`, etc.
+  and refer the target build path (*.apk or *.app) at `Build path to test` input.
+website: https://github.com/autifyhq/bitrise-step-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-step-autify-upload
+support_url: https://github.com/autifyhq/bitrise-step-autify-upload/issues
+published_at: 2022-08-30T17:45:53.952815-07:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 98793af6906f0d16bfa9ebc853eb10e3b426fa90
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- access_token: $AUTIFY_MOBILE_ACCESS_TOKEN
```

Comment:
Please remove the default env var value.

Env var should be used as a default value, in case the given env var has an initial value during a Bitrise build. These are the env vars set by the bitrise.io website, by the Bitrise CLI, or by Steps (as Step Output env var).

A required input with a default value is not highlighted in the Workflow Editor, so users might miss configuring it.

Reply:
Addressed https://github.com/autifyhq/bitrise-step-autify-upload/pull/28